src/screens/ManageTag/ManageTag.tsx

```
@@ -165,30 +165,21 @@
+        after: userTagAssignedMembersData?.tag.assignees.pageInfo.endCursor,
       },
       updateQuery: (
-        prevResult: { getAssignedUsers: InterfaceQueryUserTagsAssignedMembers },
-        {
-          fetchMoreResult,
-        }: {
-          fetchMoreResult: {
-            getAssignedUsers: InterfaceQueryUserTagsAssignedMembers;
-          };
-        },
+        prevResult: { tag: any },
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)